src/Component/Style/Style.tsx

```diff
@@ -276,6 +276,49 @@ export class Style extends React.Component<StyleProps, StyleState> {
     this.setState({ruleGeneratorWindowVisible: false});
   }
 
+  disableMenu = (name: string, rowKeys: number[]): boolean => {
```

Comment:
A small documentation would be nice.

Reply:
Done in https://github.com/terrestris/geostyler/pull/632/commits/efabb8fe2e75965d190e9cf62a4e9f1016d1cd31